src/components/actions/Button/Button.tsx

```diff
@@ -19,7 +19,7 @@ export interface CubeButtonProps extends CubeActionProps {
     | 'outline'
     | 'neutral'
     | string;
-  size?: 'small' | 'default' | 'large' | string;
+  size?: 'small' | 'default' | 'large' | (string & {});
```

Comment:
looks like a dirty hack

Reply:
Indeed, this hack enables autocomplete for the keywords in typescript. See more in: https://github.com/microsoft/TypeScript/issues/29729